test/views/declaration/TotalPackageQuantityViewSpec.scala

```diff
@@ -70,6 +70,11 @@ class TotalPackageQuantityViewSpec extends UnitViewSpec with ExportsTestData wit
           saveButton.text() must be("site.save_and_continue")
         }
 
+        "display Tariff section text" in {
+          val tariffText = view.getElementsByClass("govuk-details__summary-text").first().text()
+          tariffText.text() must be("site.details.summary_text_this")
```

Comment:
please add test for having tanslation key - first test case in suite

Reply:
done